src/Parser/BrewmasterMonk/Modules/Features/CastEfficiency.js

```diff
@@ -0,0 +1,65 @@
+import SPELLS from 'common/SPELLS';
+import CoreCastEfficiency from 'Parser/Core/Modules/CastEfficiency';
+
+const SPELL_CATEGORY = {
+  ROTATIONAL: 'Rotational Spell',
+  COOLDOWNS: 'Cooldown',
+  UTILITY: 'Utility',
+};
+
+class CastEfficiency extends CoreCastEfficiency {
+  static CPM_ABILITIES = [
```

Comment:
It would be nice if *all* abilities were here in one form or another. As an example look at the Disc Priest CastEfficiency which has *rotational spell*, *cooldowns*, *spells* and *utility* categories. It even has Shackle Undead so that the list is complete.